OpenRA.Game/WRot.cs

```diff
@@ -29,6 +29,27 @@ public struct WRot : IEquatable<WRot>
 		public static WRot operator -(WRot a, WRot b) { return new WRot(a.Roll - b.Roll, a.Pitch - b.Pitch, a.Yaw - b.Yaw); }
 		public static WRot operator -(WRot a) { return new WRot(-a.Roll, -a.Pitch, -a.Yaw); }
 
+		public static WRot operator *(WRot a, WRot b)
```

Comment:
We should be really careful when applying this operator, since multiplying rotations is non-commutative! I think at least a comment to that effect here would be in order because I can imagine a lot of potential frustration when people aren't aware of that when trying to use it. We should also double check the applications in the second commit.

Reply:
The second commit accounts for the non-commutativity, but good idea to add a comment here.